libuvc/src/main/jni/UVCCamera/UVCPreviewBase.cpp

```diff
@@ -119,25 +116,23 @@ inline const bool UVCPreviewBase::isRunning() const { return mIsRunning; }
 int UVCPreviewBase::setPreviewSize(int width,
                                    int height,
                                    int fps,
-                                   int mode,
```

Comment:
Hi @vshcryabets, indeed, if this is a requirement, we could put back a `format` parameter (clearer than `mode`  imho), which takes one of `uvc_frame_format` enumerate. By default this value could be set to the ANY enum.

Reply:
Yes, that would be great. uvc_frame_format - exactly what should be there.

Reply:
Hi @vshcryabets , I have been thinking a bit more about this.

This code is related to camera preview, and I think the frame format doesn't matter much for the preview, as long at the frames are correctly displayed on screen.

In the other hand, it would be interesting to be able to chose the frame format for the video recording use case. What do you think?